README.md

```diff
@@ -134,6 +134,11 @@ class CreateSomeRangeRecord < ActiveRecord::Migration[5.1]
        start_range: "2019-06-08",
        end_range: "2019-06-09"
   end
+
+  def down
+    drop_table :some_range_records
+    drop_table :some_range_records_template
+  end
```

Comment:
Not sure how I feel about adding this, since the `down` method isn't a pattern used by everyone. Also, if we add it here, it feels like we should add it to _all_ of the migration examples, which seems redundant

Reply:
Ok, removed this code